libraries/AP_Common/Location.cpp

```diff
@@ -376,14 +376,14 @@ bool Location::sanitize(const Location &defaultLoc)
 assert_storage_size<Location, 16> _assert_storage_size_Location;
 
 
-// return bearing in centi-degrees from location to loc2
-int32_t Location::get_bearing_to(const struct Location &loc2) const
+// return bearing in radians from location to loc2, return is 0 to 2*Pi
+ftype Location::get_bearing(const struct Location &loc2) const
```

Comment:
```suggestion
ftype Location::get_bearing_rad(const struct Location &loc2) const
```

I think we've got a policy of degrees in our interfaces, radians internally.  So if we're returning radians it should be in the name...

Reply:
that would break existing scripts that use that API. If alias worked properly we could use that, but for now we're stuck with this interface